services/workflows/Dockerfile

```diff
@@ -2,7 +2,7 @@
 ARG GO_VERSION
 ARG UPSTREAM_REPO
 ARG UPSTREAM_TAG
-FROM golang:${GO_VERSION:-1.16.4} AS builder
+FROM golang:${GO_VERSION:-1.18} AS builder
```

Comment:
I think we're ok to remove the variable here (and the ARG in line 1), and just call it like
`FROM golang:1.18-alpine AS builder`

Reply:
Updated to `FROM golang:1.18-alpine3.16 AS builder`